docs/pages/[id].js

```diff
@@ -0,0 +1,65 @@
+// @flow strict
```

Comment:
can we add a top what/why/how comment on this page?

idk, something like

//* 
`pages/[id].js` renders markdown, statically generated files. 
Pages files must be within ./markdown folder 
*//